Robot2019/src/main/java/frc/robot/RobotMap.java

```diff
@@ -77,7 +78,7 @@
     rightEnc = new Encoder(new DigitalInput(2), new DigitalInput(3));
 
     // Initialize lights
-    lights = new VictorSP(3); // TODO: Set this to correct port
+    lights = new Relay(3); // TODO: Set this to correct port
```

Comment:
Might as well make this port 0 since it's the only relay we're using. Just tell Stephen that's where to connect.